src/pages/Home/index.tsx

```diff
@@ -15,9 +18,13 @@ export const Home = () => {
   // }, []);
 
   return (
-    <div>
-      <h2>Home</h2>
-      {!userData && <SignIn />}
+    <div className="splash-page">
+      <div className="ni-wa-container">
+        <Niwa className="ni-wa-character" />
+        <h1 className="ni-wa">ni•wa</h1>
+        <h3 className="ni-wa-definition">a garden or courtyard</h3>
+        {!userData && <SignIn />}
+      </div>
     </div>
```

Comment:
Its ok to do it in a different PR,
I think we should show in the home page:
- show an explanation of how to get started and whats my niwa also when the user is not logged in 
- and show the list of gardens and a add new garden button when user is logged in

Reply:
Sounds good!